src/autosuspend/checks/activity.py

```diff
@@ -94,23 +94,47 @@ def check(self):
 
 class Kodi(NetworkMixin, Activity):
 
-    QUERY = '?request={"jsonrpc": "2.0", "id": 1, ' \
-            '"method": "Player.GetActivePlayers"}'
+    @classmethod
+    def collect_init_args(cls, config) -> Dict[str, Any]:
+        try:
+            args = NetworkMixin.collect_init_args(config)
+            args['suspend_while_paused'] = \
+                config.getboolean('suspend_while_paused', fallback=False)
```

Comment:
Can you break this line after opening bracket? That removes the need for the trailing slash.

Reply:
Fixed.